src/views/Voting/components/CastVoteModal/MainView.tsx

```diff
@@ -41,8 +42,12 @@ const MainView: React.FC<MainViewProps> = ({
         <Text color="secondary" mb="8px" textTransform="uppercase" fontSize="12px" bold>
           {t('Your Voting Power')}
         </Text>
-        {isLoading ? (
+        {isLoading && !isError ? (
           <Skeleton height="64px" mb="24px" />
+        ) : isError ? (
+          <Text mb="12px" color="red">
+            {t('Error fetching Voting Power')}
```

Comment:
Hmm. The error is a good indicator but it's not informative. As a user, I don't know what the error I'm facing and what the next action I should take to resolve it.

Reply:
Yeah, useSWR will retry to fetch automatically. (maybe we can tell the user will retry to fetch automatically?)
And about the wording. I am discussing this with Chef Yuna, and she said she wants to reword this message.